### PR TITLE
refactor: add zh_CN readme template

### PR DESCRIPTION
**Pull request by fengmk2**
https://github.com/eggjs/egg/issues/34#issuecomment-269418718

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by atian25** [comment]
+1.

docs 而不是 refactor 吧?

**Comment by popomore** [comment]
README.md 不改？

File: boilerplate/README.zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+-->
+
+## 依赖说明
+
+### 依赖的 egg 版本
+
+egg-{{name}} 版本 | egg 1.x
+--- | ---
+1.x | 😁
+0.x | ❌
```
Comment (popomore) [inline boilerplate/README.zh_CN.md]: 不要 0.x 了？

**Comment by fengmk2** [comment]
嗯，改

File: boilerplate/README.zh_CN.md (hunk 1 above)
Comment (fengmk2) [inline boilerplate/README.zh_CN.md]: 这是一个示例

**Comment by popomore** [comment]
有些提示可以用 html 注释，就像我们 issue template 那样

**Comment by fengmk2** [comment]
好的

**Pull request merged**